docs/dev/annual_updates.rst

```diff
@@ -142,20 +145,27 @@ B. FERC Form 1
 This is necessary to enable mapping associations between the FERC 1 and EIA plants and
 utilities later.
 
-**3.B.2)** You can use the ``devtools/ferc1-etl-debug.ipynb`` notebook to run the
-extract process for each table.
+**3.B.2)** Like EIA, you can either materialize the raw assets in Dagster or
+use the ``devtools/ferc1-etl-debug.ipynb`` notebook to run the extract process for
+each table.
+
+C. EPA CEMS
+^^^^^^^^^^^
+**3.C.1)** The CEMS data are so large that it doesn't make sense to store a raw and
+cleaned version of the data in the database. We'll test the extraction and
+transformation steps together in the next section.
 
 4. Update Table & Column Transformations
 ----------------------------------------
 Currently, our FERC and EIA tables utilize different transform processes.
 
 A. EIA Forms
 ^^^^^^^^^^^^
-**4.A.1)** Use the EIA ETL Debugging notebook mentioned above to run the initial
-transform step on all tables of the new year of data and debug any failures. If any new
-tables were added in the new year of data you will need to add a new transform function
-for the corresponding dataframe. If new columns have been added, they should also be
-inspected for cleanup.
+**4.A.1)** Either materialize the clean and/or normalized dagster asset groups or use
```

Comment:
depending on how shortly this renaming process will be (i.e., if we think it will happen before the final release in october) I can probably go ahead and just put the name `_core` in here right now.

Reply:
Yes I hope it'll all be in before October. @bendnorman and I are currently working on it, he might have a more realistic sense of timelines.

Reply:
ok I updated it, lmk what you think!